### ISSUE TITLE
 NXgroup below NXroot is not a valid base class

### ISSUE THREAD

**Issue by jkotan** [nexpy/nxvalidate#41]
According to https://github.com/nexusformat/definitions/issues/1429 it is allowed to create multi-format files with groups below NXroot without the NX_class attribute defined. However, when I do it I get the following error:
```
NXroot: /
    NXgroup: /nexus_logs
        NXgroup is not a valid base class
    NXentry: /scan
        NXinstrument: /scan/instrument

``` 
It would be good to not show it as an error,  i.e. at least on the NXroot level.
Also the error message is not  really clear i.e. I have not defined any group of NXgroup  type

**Comment by rayosborn** [comment]
Thanks for raising the issue. It should be fairly easy to fix.

**Comment by rayosborn** [comment]
This should have been fixed by PR #43.

**Comment by jkotan** [comment]
Thank you. Indeed, it fixes the issue.
